### DISCUSSION TITLE
Updating Wake Model Parameters

### DISCUSSION THREAD

**Discussion by sachinkumawat1** [NatLabRockies/floris#941] [General]
Hi, i am working on a project in which i have to perform sensitivity analysis for which i have to check the model output for different model parameters. but i am unable to figure out how to update new values to the model parameters.

**Comment by misi9170** [comment]
Hi @sachinkumawat1 ,

The wake model parameters can be updated directly in the floris input yaml files (e.g. by changing the [parameters of the GCH model](https://github.com/NREL/floris/blob/main/examples/inputs/gch.yaml#L226)) or by using the [`set_param()` method of the `FlorisModel`](https://github.com/NREL/floris/blob/main/floris/floris_model.py#L1515). A third option is to export convert an instantiated `FlorisModel` object as a dictionary (using `fmodel_dict = fmodel.core.as_dict()`); altering the dictionary keys directly; and reinstantiating the model as `fmodel = FlorisModel(fmodel_dict)`.

**Comment by sachinkumawat1** [comment]
@misi9170 thank you!!
